### ISSUE TITLE
Provide an `of_articletype_or` scope for Goldencobra::Article

### ISSUE THREAD

**Issue by 5minpause** [ikuseiGmbH/Goldencobra#85]
There is this scope: https://github.com/ikuseiGmbH/Goldencobra/blob/master/app/models/goldencobra/article.rb#L126

``` ruby
scope :articletype, lambda{ |name| where(article_type: name)}
```

We need a second scope, that uses `OR` for scoping.

**Comment by marcometz** [comment]
use case would be something like this? 
`Goldencobra::Article.articletype(["Report Index", "Event Show"], any: true)`

**Comment by 5minpause** [comment]
Yes, something like this. Do you want to create a Pull Request for this? 😄
